.github/workflows/e2e.yaml

```diff
@@ -1,9 +1,8 @@
 name: Dozer E2E Test
 
 on:
-  pull_request_target:
-    branches: [main]
-  workflow_dispatch:
+  schedule:
```

Comment:
Don't need to run on `merge_group` if we're running nightly? And `workflow_dispatch` can be kept so it's easier to test